media-sound/sinus/patches/sinus-0.1.3.patchset

```diff
@@ -0,0 +1,53 @@
+From 3ac8e8243d63bef82ce7d10fd734e5412088535b Mon Sep 17 00:00:00 2001
+From: Bach Nguyen <bach5000@gmail.com>
+Date: Thu, 22 Nov 2018 09:19:43 +0000
+Subject: Fix build
+
+
+diff --git a/sinus.c b/sinus.c
+index aaab435..b7cd9e4 100644
+--- a/sinus.c
++++ b/sinus.c
+@@ -235,29 +235,26 @@ void wave_to_little_endian(struct wave_t *wave)
+ #ifdef __LITTLEENDIAN__
+   return;
+ #else
+-  int i;
+-  void *p;
+ 
+-  switch(wave->point_size)
++int i = 0;
++switch(wave->point_size)
+   {
+     case 1:
+       return;
+     case 2:
+-      p = wave->data;
+-      for(i=0; i<wave->data_size; i+=2)
++      for(INT16 *p = wave->data; i < wave->data_size; i += 2)
```

Comment:
```diff
-+      for(INT16 *p = wave->data; i < wave->data_size; i += 2)
++      for(p = wave->data; i < wave->data_size; i += 2)
```
(gcc2 :)

Reminder: you need to edit `sinus.c` and then regenerate the patchset.